### ISSUE TITLE
Add option to specify colors for point colorer as array of objects containing r,g,b, and opacity

### ISSUE THREAD

**Issue by joshua-gould** [PAIR-code/scatter-gl#31]
This makes plotting lots of points significantly faster because colors do not have to be parsed

**Comment by cannoneyed** [comment]
Yeah, this is a real annoying pain point - in fact, I tried making a little system for this but wound up getting frustrated by some of the ways we handled colors internally (hint, it's all very tied to THREE)... anyway, I think this is worth revisiting.
